js/main.js

```diff
@@ -20,14 +30,82 @@ function handleSubmit(event) {
   event.preventDefault();
   const newFormEntry = {
     entryId: data.nextEntryId,
-    nextEntryId: data.nextEntryId + 1,
     title: $entryTitle.value,
     photoURL: $photoURL.value,
     notes: $entryNotes.value,
   };
   data.nextEntryId += 1;
   data.entries.unshift(newFormEntry);
+  if (data.entries.length > 0) {
+    toggleNoEntries();
+  }
+  window.location.reload();
   $img?.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form?.reset();
   writeData();
+  renderEntry(data.entries[data.nextEntryId]);
+  viewSwap('entries');
 }
+document.addEventListener('DOMContentLoaded', () => {
+  const storedData = readData();
```

Comment:
You should not be calling `readData` in this function. You should just be utilizing `data.entries` directly